src/anchos.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+const Web3 = require('web3');
+const Mosaic = require('@openstfoundation/mosaic.js');
+const program = require('commander');
+const fs = require('fs');
+
+const StateRootAnchorService = require('./state_root_anchor_service.js');
+
+program
+  .version('0.1.0', '-v, --version')
+  .option('-d --direction <origin | auxiliary>', 'The source chain to anchor a state root '
+  + 'to the target chain.', /^(origin|auxiliary)$/i)
+  .option('-d, --delay <block number>', 'The delay to wait in blocks on the source '
+  + 'chain before anchoring.', /^\d+$/i)
+  .option('-c, --config <config-file-path>', 'The config file path to read chains '
+  + 'access info and contract addresses from.')
+  .parse(process.argv);
+
+// Read chains access info and contract addresses from configuration file.
+const config = JSON.parse(fs.readFileSync(program.config, 'utf8'));
```

Comment:
you can use `require` on a json file

Reply:
I would rather recommend not to do it dynamically. Plus eslint complains also :) (AirBnb)

  `Calls to require() should use string literals. lint(import/no-dynamic-require)`

Reply:
Agreed, thought the config path was constant. Sorry.